packages/caravan-fees/src/rbf.ts

```diff
@@ -143,14 +143,14 @@ export const createCancelRbfTransaction = (
     if (
       newTxTemplate.feeRateSatisfied &&
       new BigNumber(newTxTemplate.currentFee).gte(
-        BigNumber.max(newTxTemplate.targetFeesToPay, txAnalyzer.minimumRBFFee),
+        BigNumber.max(newTxTemplate.targetFeesToPay, txAnalyzer.fee),
```

Comment:
This points to some of what I find confusing about the naming. What is the `fee` property on the analyzer again? Reading below, maybe it should be originalFee?

Reply:
So basically this conditional is saying, “if the templates target fee rate is satisfied and absolute fees are greater than the larger of either target fees of template (which in this case is rbf target) OR the original fees of the transaction, then the template is good to generate a psbt from”. What if you just set the target fee to be the max of either the rbf value OR the original? Then you just have to check if fees are satisfied right? Second question: what is the original fees from? Is that from the transaction that’s being bumped? Don’t the RBF rules mean that the rbf target fees ALWAYS have to be larger than the original tx fees?

maybe the comment below shouldn’t say “based on desired rate by user”, since in this case it’s based on rbf calculations right?

Reply:
> What if you just set the target fee to be the max of either the rbf value OR the original? Then you just have to check if fees are satisfied right?

Actually the `newTxTemplate.targetFeesToPay` is  dynamic as it is calculated on based on `current Template size *  targetFeeRate `  ...  

`currentFee` is based on the actual inputs and outputs in the transaction, while `targetFeesToPay` is calculated based on the target fee rate and estimated transaction size.

so we cannot set it to be  max of either the rbf value OR the original 

> Second question: what is the original fees from? Is that from the transaction that’s being bumped? Don’t the RBF rules mean that the rbf target fees ALWAYS have to be larger than the original tx fees?

Yes it's from the original tx ...